hazelcast/src/test/java/com/hazelcast/map/impl/nearcache/NearCacheTest.java

```diff
@@ -1090,4 +961,30 @@ public void smoke_near_cache_population() {
         // 5. assert number of entries in client Near Cache
         assertEquals(mapSize, ((NearCachedMapProxyImpl) nearCachedMap).getNearCache().size());
     }
+
+    private void verifyNoInvalidationsWith(BiConsumer<IMap<Integer, Integer>, Integer> operation) {
+        String mapName = randomMapName();
+
+        Config config = getConfig();
+        config.getMapConfig(mapName).setNearCacheConfig(newNearCacheConfig().setInvalidateOnChange(true));
+
+        TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
+        HazelcastInstance[] instances = factory.newInstances(config);
+
+        IMap<Integer, Integer> map = instances[0].getMap(mapName);
+
+        int entryCount = 100;
+
+        populateMap(map, entryCount);
+        populateNearCache(map, entryCount);
+
+        NearCacheStats stats = getNearCacheStats(map);
+
+        long invalidationsBefore = stats.getInvalidations();
+
+        operation.accept(map, entryCount);
+
+        long invalidationsAfter = stats.getInvalidations();
+        assertEquals(invalidationsBefore, invalidationsAfter);
```

Comment:
Minor: a little bit waiting can make test more reliable.

```suggestion
        assertTrueAllTheTime(() -> {
            long invalidationsAfter = stats.getInvalidations();
            assertEquals(invalidationsBefore, invalidationsAfter);
        }, 3);
```

Reply:
Makes sense, I have applied this to both of the tests